trailblazer/constants.py

```diff
@@ -170,3 +170,12 @@ class TrailblazerStatusColor(StrEnum):
 }
 
 PIPELINES = [pipeline.lower() for pipeline in Pipeline]
+
+
+class JobType(StrEnum):
+    UPLOAD: str = "upload"
+    ANALYSIS: str = "analysis"
+
+    @classmethod
+    def types(cls) -> tuple:
+        return tuple(cls)
```

Comment:
Is there no builtin functionality for this?

Reply:
Feel free to keep it as is, but I don't find it clearer than `tuple(JobType)`

Reply:
We have these helper methods on other enums so I'll keep it. But we might want to change that pattern.